pkg/machines/libvirt.es6

```diff
@@ -226,10 +226,16 @@ function spawnVirshReadOnly({connectionName, method, name, failHandler}) {
 function parseDumpxml(dispatch, connectionName, domXml) {
     const xmlDoc = $.parseXML(domXml);
 
+    if (!xmlDoc) {
+        console.error(`Can't parse dumpxml, input: "${domXml}"`);
+        return ;
+    }
```

Comment:
This looks like it would silently ignore parsing errors. I don't claim to understand the details of what's going on here, but this looks a bit dubious? @stefwalter , are you familiar with this part?